jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/internal/ByteChannelContentSource.java

```diff
@@ -11,7 +11,7 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.io.content;
```

Comment:
This was a public class.
Are we ok with breaking compat?

Reply:
@joakime it was only public for a few days.  It has never been released.